ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -474,6 +476,12 @@ public static Hive get() throws HiveException {
     return get(true);
   }
 
+  public static Hive get(IMetaStoreClient msc) throws HiveException, MetaException {
```

Comment:
If this is only used for testing purposes would be good to annotate this method with @VisibleForTesting

Reply:
Done.